packages/backend/src/open_payments/payment/incoming/service.test.ts

```diff
@@ -596,4 +596,37 @@ describe('Incoming Payment Service', (): void => {
       })
     })
   })
+
+  describe('addReceivedAmount', (): void => {
+    let incomingPayment: IncomingPayment
+
+    beforeEach(
+      async (): Promise<void> => {
+        const incomingPaymentOrError = await incomingPaymentService.create({
+          accountId,
+          description: 'Test incoming payment',
+          incomingAmount: {
+            value: BigInt(123),
+            assetCode: asset.code,
+            assetScale: asset.scale
+          },
+          expiresAt: new Date(Date.now() + 30_000),
+          externalRef: '#123'
+        })
+        assert.ok(!isIncomingPaymentError(incomingPaymentOrError))
+        incomingPayment = incomingPaymentOrError
+      }
+    )
+
+    test('throws if no TB account found', async (): Promise<void> => {
```

Comment:
This test could just be moved into the `describe('Create/Get IncomingPayment`
You can call `createPayment` at the start of it.

Reply:
:+1: I had it there first and then moved it :sweat_smile: